presto-main/src/main/java/com/facebook/presto/operator/HttpTaskResultClient.java

```diff
@@ -85,10 +85,10 @@
 import static java.util.concurrent.TimeUnit.SECONDS;
 
 @ThreadSafe
-public final class HttpPageBufferClient
+public final class PageBufferClient
```

Comment:
The file rename in this commit renames "HttpPageBufferClient.java" to `HttpTaskResultClient.java", while the class name is `PageBufferClient`?  Is this file `HttpTaskResultClient.java" intended to be added in this commit?

Reply:
I guess the rebase messes the following commit to this one...